src/deck.jsx

```diff
@@ -49,37 +50,56 @@ class Deck extends React.Component {
   }
   _handleKeyPress(e) {
     const event = window.event ? window.event : e;
-    if (event.keyCode === 37) {
+    if (event.keyCode === 37) { // left arrow
       this._prevSlide();
     }
-    if (event.keyCode === 39) {
+    if (event.keyCode === 39) { // right arrow
       this._nextSlide();
     }
+    if (event.keyCode === 79) { // o
+      this._toggleOverviewMode();
+    }
+    if (event.keyCode === 80) { // o
+      this._togglePresenterMode();
+    }
+  }
+  _toggleOverviewMode() {
```

Comment:
Ok so requiring a keying prop works magically. But requiring a key sucks, so I'm going to see if I can find any way of doing that without requiring the user to add a key.